Blockzilla/SettingsViewController.swift

```diff
@@ -584,14 +607,14 @@ class SettingsViewController: UIViewController, UITableViewDataSource, UITableVi
     @objc private func whatsNewClicked() {
         highlightsButton?.tintColor = UIColor.white
         
-        guard let url = SupportUtils.URLForTopic(topic: "whats-new-focus-ios-7") else { return }
```

Comment:
Should not change this in this PR. One PR for one issue. changing the support URL is not included in adding a search toggle.